servers/rendering/renderer_rd/shaders/effects/fsr2/SCsub

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+Import("env")
+
+if "RD_GLSL" in env["BUILDERS"]:
+    # find all include files
```

Comment:
```suggestion
    # Find all include files.
```

Reply:
I can fix the comment style here, but it's worth pointing out I just copied and pasted this file from the other folders that have the same build system: https://github.com/godotengine/godot/blob/master/servers/rendering/renderer_rd/shaders/environment/SCsub

Personally I'd prefer if all of this was handled by the parent script in a recursive way, but that's how the current architecture works. It might harm the possibility of finding and replacing all of the instances of the file later if I make it different here. Should I still go ahead with changing it?

Reply:
I would prefer keeping it consistent, even if it doesn't fit the style.

This can be refactored later all at once (include possibly reducing the code duplication).